Procfile

```diff
@@ -1,3 +1,2 @@
 web: bundle exec puma -C config/puma.rb
 worker: bundle exec sidekiq -C config/sidekiq.yml
-release: bundle exec rake db:migrate
```

Comment:
`release` is used to automatically run migrations on deployment for heroku, so I don't really want to remove that line if possible

Reply:
Ok, see the point. Sorry, I'm really a Heroku noop. 

So I recommend to use a different Procfile for foreman. Let me adapt the PR ...